item_type.cpp

```diff
@@ -391,7 +391,7 @@ ItemAttributes ItemTypes::Poem::getAttributes(const ContentFactory*) const {
       i.itemClass = ItemClass::SCROLL;
       i.weight = 0.1;
       i.modifiers[AttrType::DAMAGE] = -10;
-      i.effect = Effect(Effects::Area{10, Effects::Filter{FilterType::ENEMY, Effects::IncreaseMorale{-0.1}}});
+      // i.effect = Effect(Effects::Area{10, Effects::Filter{FilterType::ENEMY, Effects::IncreaseMorale{-0.1}}});
```

Comment:
https://media.discordapp.net/attachments/684153932817236074/697561732851171438/unknown.png

Don't know what syntax to use.